Authentication/Login/View/LoginViewDelegate.swift

```diff
@@ -65,20 +64,20 @@ public final class DefaultLoginViewDelegate: LoginViewDelegate {
         configureErrorElements(loginView)
     }
     
-    private func configureMainButton(loginView: LoginViewType) {
+    fileprivate func configureMainButton(_ loginView: LoginViewType) {
```

Comment:
`func configureMainButton(in loginView: LoginViewType)` or `func configureMainButton(from loginView: LoginViewType)` ?